spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/actions/TestRemoveOrphanFilesAction.java

```diff
@@ -150,10 +150,20 @@ public void testDryRun() throws IOException, InterruptedException {
     Assert.assertEquals("Action should find 1 file", invalidFiles, result2.orphanFileLocations());
     Assert.assertTrue("Invalid file should be present", fs.exists(new Path(invalidFiles.get(0))));
 
+    String actualFilesTableName = "actualFilesTable";
+    spark.createDataset(allFiles, Encoders.STRING()).toDF("file_path").createOrReplaceTempView(actualFilesTableName);
     DeleteOrphanFiles.Result result3 = actions.deleteOrphanFiles(table)
+        .deleteWith(s -> { })
+        .actualFilesTable(actualFilesTableName)
+        .execute();
+
+    Assert.assertEquals("Action should find 1 file", invalidFiles, result3.orphanFileLocations());
+    Assert.assertTrue("Invalid file should be present", fs.exists(new Path(invalidFiles.get(0))));
+
+    DeleteOrphanFiles.Result result4 = actions.deleteOrphanFiles(table)
```

Comment:
If we add support for things like `older_than`, we should probably make a separate test case for the actual files table option so we can exercise various scenarios focused on that one area.

Reply:
Agreed. We usually want to add separate test cases anyway.